Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -43620,6 +43621,26 @@
 			shellPath = /bin/sh;
 			shellScript = "SRC_DIR=\"$SRCROOT/Modules/modern-media-controls\"\nDST_DIR=\"$BUILT_PRODUCTS_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH/modern-media-controls\"\n\nmkdir -p \"$DST_DIR\"\n\nif [ \"${WK_PLATFORM_NAME}\" == \"macosx\" ]; then\n    IMG_OS_PREFIX=\"macOS\"\nelse\n    if [[ \"${WK_PLATFORM_NAME}\" == *\"watch\"* ]]; then\n        IMG_OS_PREFIX=\"watchOS\"\n    else\n        IMG_OS_PREFIX=\"iOS\"\n    fi\nfi\n\nif [ -n \"$IMG_OS_PREFIX\" ]; then\nrsync -aq --exclude \".svn\" --exclude \".DS_Store\" \"$SRC_DIR/images/$IMG_OS_PREFIX/\" \"$DST_DIR/images\"\nfi\n\ncd \"$SRC_DIR\"\n";
 		};
+		947D341A2BA12DD50051D6D0 /* Remove stale PALSwift.h added in 275246@main */ = {
+			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
Now that you have an output edge, this should be removed so that the build system tracks it appropriately.
```suggestion
			alwaysOutOfDate = 0;
```

Will remove the message in build logs like:
> Run script build phase 'Remove stale PALSwift.h added in 275246@main' will be run during every build because the option to run the script phase "Based on dependency analysis" is unchecked.

Reply:
done.